lib/elixir/lib/integer.ex

```diff
@@ -217,16 +218,16 @@ defmodule Integer do
   """
   @spec undigits([integer], pos_integer) :: integer
   def undigits(digits, base \\ 10) when is_list(digits) and is_integer(base) and base >= 2 do
-    do_undigits(digits, base, 0)
+    undigits(digits, base, 0)
   end
 
-  defp do_undigits([], _base, acc), do: acc
+  defp undigits([], _base, acc), do: acc
 
-  defp do_undigits([digit | _], base, _) when is_integer(digit) and digit >= base,
```

Comment:
I think this guard is mandatory if you want to ensure the function signature?

Reply:
You're right. It's more consistent to keep this as is.